src/main/java/it/pn/frontend/e2e/pages/mittente/AutorizziInvioDatiPAPage.java

```diff
@@ -13,7 +13,7 @@ public class AutorizziInvioDatiPAPage extends BasePage {
     private static final Logger logger = LoggerFactory.getLogger("AutorizziInvioDatiPAPage");
 
     @FindBy(css = "button[name='confirm']")
```

Comment:
inserire in documento id

Reply:
li riguarda il bottone conferma di selfcare era già messo nel primo documento